tests/src/app/testqgsidentifyresultsdialog.cpp

```diff
@@ -0,0 +1,270 @@
+/***************************************************************************
+     testqgsidentifyresultsdialog.cpp
```

Comment:
There's other tests for the dialog in the identify map tool tests. Maybe we should put these new ones in there, and then rename to just testqgsidentify.cpp?

Reply:
done